src/Passes/Source/RemoveDisallowedAttribubtesPass/RemoveDisallowedAttribubtesPass.hpp

```diff
@@ -0,0 +1,69 @@
+#pragma once
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include "Types/Types.hpp"
+
+#include "Llvm/Llvm.hpp"
+
+#include <functional>
+#include <unordered_set>
+#include <vector>
+
+namespace microsoft
+{
+namespace quantum
+{
+
+    class RemoveDisallowedAttribubtesPass : public llvm::PassInfoMixin<RemoveDisallowedAttribubtesPass>
```

Comment:
Consider fixing a typo in the class and file names (syntax typos are highlighted in VSCode).
```suggestion
    class RemoveDisallowedAttributesPass : public llvm::PassInfoMixin<RemoveDisallowedAttributesPass>
```

Reply:
Fixed.